CMakeLists.txt

```diff
@@ -346,7 +346,7 @@ endif()
 if(WIN32 OR ARM OR PPC64LE OR PPC64 OR PPC)
   set(OPT_FLAGS_RELEASE "-O2")
 else()
-  set(OPT_FLAGS_RELEASE "-Ofast")
+  set(OPT_FLAGS_RELEASE "-O3")
```

Comment:
clang recommends to replace `-Ofast` with `-O3 -ffast-math`:
> Users are advised to switch to ``-O3 -ffast-math`` if the use of non-standard math behavior is intended, and ``-O3`` otherwise.

I don't think this PR should change the current behavior which enables fast math.

Reply:
> clang recommends to replace `-Ofast` with `-O3 -ffast-math`:
> 
> > Users are advised to switch to `-O3 -ffast-math` if the use of non-standard math behavior is intended, and `-O3` otherwise.
> 
> I don't think this PR should change the current behavior which enables fast math.

I see, I noticed that there is a conflicting opinion between [SChernykh](https://github.com/SChernykh) and [0xFFFC0000](https://github.com/0xFFFC0000). 

As of now, I have removed the lines as suggested by [0xFFFC0000](https://github.com/0xFFFC0000). Should I add the flag -ffast-math instead?